exist-core/src/main/java/org/exist/xquery/Cardinality.java

```diff
@@ -48,7 +48,7 @@ public enum Cardinality {
     EXACTLY_ONE(ONE),
 
     //TODO(AR) eliminate this in favour of probably ONE_OR_MORE
-    _MANY(MANY),
```

Comment:
I put the `_` here to denote that this is a private internal value that should only be used inside eXist-db's internals. This should not be renamed. As the `TODO` states, if anything this should be completely removed.

Reply:
> Do you recall what getting rid of them entails?

Yes fixing the weird code that uses it, its just a few places. You can find it via Find References in IntelliJ.

> Which means that up until a certain point in the not so distant past this was changed in exist.

Actually this was done almost 6 years ago now - so definitely the distant past!